tests/Unit/Filters/Frame/CustomFrameFilterTest.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Tests\FFMpeg\Unit\Filters\Frame;
+
+use FFMpeg\Filters\Frame\CustomFrameFilter;
+use Tests\FFMpeg\Unit\TestCase;
+
+class CustomFrameFilterTest extends TestCase
+{
+    public function testApplyCustomFrameFilter()
+    {
+	$frame = $this->getFrameMock();
+
+	$filter = new CustomFrameFilter('whatever i put would end up as a filter');
```

Comment:
Please fix the formatting(fix intending).

Reply:
Done. Sorry for inconvenience.